flow/project/cli/config/config.go

```diff
@@ -118,6 +118,8 @@ func (c *Contracts) GetByNameAndNetwork(name string, network string) Contract {
 	return contracts[0]
 }
 
+// REF: this filtering can cause error if not found, better to refactor to returning
```

Comment:
I would turn your `REF`s into `TODO`s, since most of these notes refer to future work/improvements